File: src/main/java/com/status/domain/Status.java
package com.status.domain;

/**
 * Created by anteastra on 20.10.2015.
 */
public class Status {

    private String ip;
    private String cpuUsage;
    private String freeMemory;
    private String totalMemory;

    public Status(String ip, String cpuUsage, String freeMemory, String totalMemory) {
        this.ip = ip;
        this.cpuUsage = cpuUsage;
        this.freeMemory = freeMemory;
        this.totalMemory = totalMemory;
    }

    public String getFreeMemory() {
        return freeMemory;
    }

    public void setFreeMemory(String freeMemory) {
        this.freeMemory = freeMemory;
    }

    public String getTotalMemory() {
        return totalMemory;
    }

    public void setTotalMemory(String totalMemory) {
        this.totalMemory = totalMemory;
    }

    public String getIp() {
        return ip;
    }

    public void setIp(String ip) {
        this.ip = ip;
    }

    public String getCpuUsage() {
        return cpuUsage;
    }

    public void setCpuUsage(String cpuUsage) {
        this.cpuUsage = cpuUsage;
    }
}

File: src/main/webapp/js/status.js
/**
 * Created by anteastra on 20.10.2015.
 */

$(document).ready(function() {

    var interval = 500;   //number of mili seconds between each call
    var refresh;
    refresh = function () {
        $.ajax({
            url: "http://localhost:8080/status"
        }).then(function (data, status, jqxhr) {
            $('.status-ip').text(data.ip);
            $('.status-cpuUsage').text(data.cpuUsage);
            $('.status-freeMemory').text(data.freeMemory);
            $('.status-totalMemory').text(data.totalMemory);
            console.log(jqxhr);
            setTimeout(function () {
                refresh();
            }, interval);
        });
    };
    refresh();
});
